src/table/virtualized-table/Body.tsx

```diff
@@ -135,7 +135,7 @@ const Body = forwardRef<BodyRef, BodyProps>((props, ref) => {
       data-key="body"
       ref={gridRef}
       innerRef={innerForwardRef}
-      style={{ overflow: 'hidden' }}
+      style={{ overflow: 'hidden', willChange: 'auto' }}
```

Comment:
May I ask what is WillChange?

Reply:
Oh interesting! I didn't know that! tnx

Reply:
Are you sure that VariableSizeGrid also needs to have WillChange?

Reply:
@LiKang6688 : this was specifically mentioned by @cbt1 , Is there any specific reason you think this may not be needed in `VariableSizeGrid `?

Reply:
as I see in the sn-pivot-table, VariableSizeGrid's WillChange is not changed. 
And it is called listStyle, it is confusing for me that it is also applied to VariableSizeGrid